assets/src/edit-story/app/media/media3p/api/media3pApiProvider.js

```diff
@@ -47,11 +47,16 @@ const Providers = {
 function Media3pApiProvider({ children }) {
   const MEDIA_PAGE_SIZE = 20;
 
-  function constructFilter(provider, searchTerm, mediaType) {
+  function constructFilter(
+    provider,
+    searchTerm,
+    selectedCategoryName,
```

Comment:
Should this be named category_id, as per the name used in the filter specification? category_id is interpreted differently depending on the provider, for example, Tenor's category id is not the name, but a 'search term' and Coverr's category may be an identifier.

Reply:
I agree. `name` is confusing. I guess I'll introduce a mapper to map from `name` of the API to a local id. That will make it more obvious.

Reply:
Tbh we should we be using the nomenclature used by our API (a category has a `name`). So we should probably rename the category_id filter key to category_name.

Reply:
I'd rather rename the API response to be `id` instead of `name` as it is quite confusing.

Reply:
I don't think we can do that, according to the AIP resources should be identified by their name, and there's no "id" in them. https://google.aip.dev/122

Reply:
According to https://google.aip.dev/122#resource-id-segments we could use the term id, but that would mean removing the `categories/` prefix that we get from the API

Reply:
Or we just encrypt it :trollface:

Reply:
I renamed everything to `categoryId`.